cranelift-wasm/src/environ/dummy.rs

```diff
@@ -201,8 +201,8 @@ impl<'dummy_environment> TargetEnvironment for DummyFuncEnvironment<'dummy_envir
 }
 
 impl<'dummy_environment> FuncEnvironment for DummyFuncEnvironment<'dummy_environment> {
-    fn is_wasm_parameter(&self, func: &ir::Function, index: usize) -> bool {
-        func.signature.params[index].purpose == ir::ArgumentPurpose::Normal
+    fn is_wasm_parameter(&self, signature: &ir::Signature, index: usize) -> bool {
+        signature.params[index].purpose == ir::ArgumentPurpose::Normal
```

Comment:
I also realized it would be nicer for embedders that this implementation be the default impl in FuncEnvironment (eg I doubt spidermonkey will require special behavior here). Can you move the code there, please?

Reply:
Done!